src/Attestation/Attestation/help/New-AzAttestation.md

```diff
@@ -14,7 +14,8 @@ Creates an attestation
 
 ```
 New-AzAttestation -Name <String> -ResourceGroupName <String> [-AttestationPolicy <String>]
- [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
Could you please add an exmple for `PolicySigningCertificateFile` here? And add a test case for `PolicySigningCertificateFile` too.

Reply:
examples are added in .md file
PolicySigningCertificateFile is a file saved in local, once the local file is removed, testcase will fail

Reply:
additional test case is added